app/services/foreman/template_snapshot_service.rb

```diff
@@ -205,6 +209,14 @@ def rocky9_dhcp
       define_host_params(host)
     end
 
+    def windows10_dhcp
+      host = FactoryBot.build(:host_for_snapshots_ipv4_dhcp_windows10,
```

Comment:
As noted earlier: this ends up with a Red Hat partition table. Perhaps this is also a problem for the Debian factories.

Reply:
I found that there is a partition table configured for Debian systems but I have not checked how thoroughly it is used. https://github.com/theforeman/foreman/blob/develop/test/factories/host_related.rb#L23